Dockerfile

```diff
@@ -17,6 +17,9 @@ LABEL permissions='\
   },\
   "HostConfig": {\
     "Binds":["/root/.config:/root/.config"],\
```

Comment:
Can we also change this to use the `extensions` folder?

I'm aware that's not relevant for this PR, but it'd be nice if we could sort that out as part of the general "fitting more with BlueOS best practices" theme :-P

Reply:
hmm... let me make a version with this pr first, as this got tested already =]